gnovm/cmd/gno/dev.go

```diff
@@ -0,0 +1,343 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"net"
+	"net/http"
+	"path/filepath"
+	"time"
+
+	"github.com/fsnotify/fsnotify"
+	"github.com/gnolang/gno/gno.land/pkg/gnoweb"
```

Comment:
I suggest we move `gnovm/cmd/gno dev` to `contribs/gnodev` (or `contribs/gno-dev`).